rootfs/scripts/usbreset.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/with-contenv bash
+# shellcheck shell=bash
+
+if [[ "$READSB_DEVICE_TYPE" == "rtlsdr" ]]; then
+    if [[ -n "$READSB_USBRESET" ]]; then
+      sleep 30
+      USBNAME=RTL2838
```

Comment:
Should `USBNAME` be defined as an environment variable in the container? Users may not be using RTL2838 radios.

For example, on my system:

```
$ lsusb | grep RTL
Bus 001 Device 014: ID 0bda:2832 Realtek Semiconductor Corp. RTL2832U DVB-T
Bus 001 Device 015: ID 0bda:2838 Realtek Semiconductor Corp. RTL2838 DVB-T
Bus 001 Device 012: ID 0bda:2838 Realtek Semiconductor Corp. RTL2838 DVB-T
Bus 001 Device 008: ID 0bda:2838 Realtek Semiconductor Corp. RTL2838 DVB-T
Bus 001 Device 005: ID 0bda:2838 Realtek Semiconductor Corp. RTL2838 DVB-T
```

We could have the environment variable `USBRESET_DEVICENAME` or similar.

Furthermore, how does this handle multiple devices?

Reply:
I haven't added multiple device support. I will look into this.

Reply:
It might be a good idea to only reset the device given in `READSB_RTLSDR_DEVICE`.